src/Campaigns.php

```diff
@@ -88,9 +88,10 @@ public function add_taxonomy_form_fields( $wp_tag ) {
 			$dropdown_args = [
 				'show_option_none' => __( 'Select Page', 'planet4-master-theme-backend' ),
 				'hide_empty'       => 0,
-				'hierarchical'     => true,
```

Comment:
I removed this because `wp_dropdown_pages` only works on hierarchical post types.

For listing pages, hierarchy doesn't make sense, the page is associated with a post, not a term.

As a workaround I made it hierarchical anyway, but we could use an alternative to `wp_dropdown_pages` that doesn't have this tight coupling.